utils/errors/errors.go

```diff
@@ -70,6 +76,13 @@ func (c customError) Cause() error {
 	return c.cause
 }
 
+func (c customError) GRPCStatus() *status.Status {
+	if c.status == nil {
+		return status.New(codes.Unknown, "error occured")
```

Comment:
do we can use c.Msg to replace the fixed string?

Reply:
I this is better if i add Error(), will do that!